acorn/src/parseutil.js

```diff
@@ -152,3 +152,15 @@ pp.isSimpleAssignTarget = function(expr) {
     return this.isSimpleAssignTarget(expr.expression)
   return expr.type === "Identifier" || expr.type === "MemberExpression"
 }
+
+pp.expectPlugin = function(pluginName) {
+  if (this.hasPlugin(pluginName)) {
+    return true
+  }
+
+  this.raise(this.pos, `Missing plugin: ${pluginName}.`)
+}
+
+pp.hasPlugin = function(pluginName) {
+  return this.plugins.includes(pluginName)
```

Comment:
We're targeting engines that don't have `includes`—so this could use `indexOf` instead.